### ISSUE TITLE
Tablescan vs. Tableseek in explain

### ISSUE THREAD

**Issue by wwar** [pingcap/tidb#14529]
## Feature Request

TiDB's EXPLAIN uses the terminology of "Tablescan" to show that the clustered index is being accessed, and "IndexScan" to show any other index is being accessed.

Whether or not this is efficient depends on the row estimate + if there is any attached range.

**Describe the feature you'd like:**

I like the SQL Server terminology of "Table Seek" and "Index Seek" for the case that the whole structure is not being read. Consider https://github.com/pingcap/tidb/issues/14525: 

I believe to the user, it looked like this is the difference between a Point Get and a Tablescan, but the second query is still very efficient:

```
TableReader_5	1.03	root	data:TableScan_4
└─TableScan_4	1.03	cop	table:sys_task_inst, range:[61099,61099], keep order:false, stats:pseudo
```

I would like it to instead say:
```
TableReader_5	1.03	root	data:TableScan_4
└─TableSeek_4	1.03	cop	table:sys_task_inst, range:[61099,61099], keep order:false, stats:pseudo
```

**Describe alternatives you've considered:**

It is somewhat fine when you are used to it.

**Teachability, Documentation, Adoption, Migration Strategy:**

A lot of people are conditioned to "table scans are bad". It is useful to prevent this conversation by calling it a Table seek. That makes TiDB slightly easier to teach.

**Comment by francis0407** [comment]
Nice suggestion. LGTM.
cc @zz-jason @qw4990

**Comment by wwar** [comment]
Fixed in #14696 (different fix, but works equally fine.)
